projects/element-ng/landing-page/si-landing-page.component.html

```diff
@@ -19,64 +22,89 @@
-            [availableLanguages]="supportedLanguages"
-          />
+    <div class="landing-page-content-inner">
+      @if (!isFullHeightSection) {
+        <div class="d-flex justify-content-between align-items-center pb-9">
+          @if (!logoUrl()) {
+            <span role="img" class="landing-page-logo"></span>
+          }
+          @if (logoUrl()) {
+            <img
+              class="landing-page-logo landing-page-custom-logo"
+              [src]="logoUrl()"
+              [alt]="heading() + '\'s Background'"
+            />
+          }
+          @if (availableLanguages && availableLanguages.length > 1) {
+            <div class="landing-page-language-switcher">
+              <si-language-switcher
+                [translationKey]="translationKey()"
+                [availableLanguages]="availableLanguages"
+              />
+            </div>
+          }
         </div>
       }
-    </div>
-    <div class="landing-page-main">
-      @if (subheading()) {
-        <div class="si-h2">{{ subheading() }}</div>
-      }
-      <h1 class="si-h1-black my-2">{{ heading() }}</h1>
-      @if (!loginAlert()) {
-        <h2 class="mt-10 si-h2">{{ subtitle() }}</h2>
-      }
-      @if (loginAlertDetails) {
-        <si-inline-notification
-          class="my-6"
-          [severity]="loginAlertDetails.severity || 'danger'"
-          [heading]="loginAlertDetails.heading"
-          [message]="loginAlertDetails.message"
-        />
-      }
-      <div class="landing-page-custom-content">
-        <ng-content />
-      </div>
-    </div>
-    <footer>
-      <div>
-        <div class="d-flex justify-content-start flex-row flex-wrap mt-6">
-          @for (link of links(); track $index) {
-            <span>
-              <a siLinkDefaultTarget="_blank" class="si-body-2 d-flex-inline me-6" [siLink]="link">
-                {{ link.title | translate }}
-              </a>
-            </span>
+      <div class="landing-page-main">
+        @if (!isFullHeightSection) {
+          @if (subheading()) {
+            <div class="si-h2">{{ subheading() }}</div>
           }
-        </div>
-        <div class="si-caption mt-4 mb-6 text-secondary d-flex flex-wrap">
-          @if (version()) {
-            <span class="me-6">{{ version() }}</span>
+          <h1 class="si-h1-black my-2">{{ heading() }}</h1>
+          <div class="position-relative">
+            <h2 class="mt-10 si-h2" [class.invisible]="loginAlert">{{ subtitle() }}</h2>
+            @if (loginAlert) {
+              <si-inline-notification
+                class="landing-page-alert w-100"
+                [severity]="loginAlert.severity || 'danger'"
+                [heading]="loginAlert.heading"
+                [message]="loginAlert.message"
+              />
+            }
+          </div>
+        }
+        <div class="landing-page-custom-content">
+          <ng-content />
+          @if (!isFullHeightSection) {
+            @if (implicitLegalAcknowledge) {
```

Comment:
I think you can directly do like this and remove additional `implicitLegalAcknowledgeTemplate` variable
`@if (typeof implicitLegalAcknowledge === 'string') {} @else {}`